docs/quick-start.md

```diff
@@ -20,15 +20,16 @@ First, install and setup cri-resource-manager.
 CRIRM_VERSION=`curl -s "https://api.github.com/repos/intel/cri-resource-manager/releases/latest" | \
                jq .tag_name | tr -d '"v'`
 source /etc/os-release
-sudo rpm -Uvh https://github.com/intel/cri-resource-manager/releases/download/v${CRIRM_VERSION}/cri-resource-manager-${CRIRM_VERSION}-0.x86_64.${ID}-${VERSION_ID}.rpm
+[ "$ID" = "sles" -o "$ID" = "opensuse-leap" ] && export ID=suse
+sudo rpm -Uvh https://github.com/intel/cri-resource-manager/releases/download/v${CRIRM_VERSION}/cri-resource-manager-${CRIRM_VERSION}-0.${ID}-${VERSION_ID}.x86_64.rpm
```

Comment:
- Works on opensuse-15.2
- Fails on Fedora-33:
```
# sudo rpm -Uvh https://github.com/intel/cri-resource-manager/releases/download/v${CRIRM_VERSION}/cri-resource-manager-${CRIRM_VERSION}-0.${ID}-${VERSION_ID}.x86_64.rpm
Retrieving https://github.com/intel/cri-resource-manager/releases/download/v0.5.0/cri-resource-manager-0.5.0-0.fedora-33.x86_64.rpm
curl: (22) The requested URL returned error: 404
error: skipping https://github.com/intel/cri-resource-manager/releases/download/v0.5.0/cri-resource-manager-0.5.0-0.fedora-33.x86_64.rpm - transfer failed
```
  would work on Fedora 33 if `s/-${VERSION_ID}//'. Maybe we're missing "-33" in uploaded filenames. Or is it wanted?